js/blocks/GraphicsBlocks.js

```diff
@@ -709,27 +717,31 @@ function setupGraphicsBlocks(activity) {
             const tur = activity.turtles.ithTurtle(activity.turtles.companionTurtle(turtle));
 
             if (args.length === 1) {
-                if (typeof args[0] === "string") {
-                    activity.errorMsg(NANERRORMSG, blk);
-                } else if (logo.inMatrix) {
-                    logo.phraseMaker.addRowBlock(blk);
-                    if (logo.pitchBlocks.indexOf(blk) === -1) {
-                        logo.pitchBlocks.push(blk);
-                    }
-                    logo.phraseMaker.rowLabels.push(activity.blocks.blockList[blk].name);
-                    logo.phraseMaker.rowArgs.push(args[0]);
-                } else if (tur.singer.inNoteBlock.length > 0) {
-                    tur.singer.embeddedGraphics[last(tur.singer.inNoteBlock)].push(blk);
+                if (args[0] > 5000) {
+                    activity.errorMsg("Back block value should be less than 5000.", blk);
```

Comment:
It would be good if we can come up with one error message that covers all cases -- less work for our translation team.

Reply:
@walterbender, should the error message just be  "Values should not exceed 5000"?

Reply:
Maybe _("Value should not exceed 5000") since we are only processing one value at a time.

Reply:
But then what about if in the XY Block the user sets both X and Y values to be exceeding 5000?

Reply:
Each value is evaluated independently, so the error message would be triggered for just one value at a time.

Reply:
@walterbender, I have made the necessary changes. Could you review it ?